File: src/main/java/com/mymedia/nagasu/dto/VideoCollectionResponse.java
package com.mymedia.nagasu.dto;

import java.util.List;

/**
 * 비디오 컬렉션(폴더) 정보를 반환하는 응답 DTO
 */
public record VideoCollectionResponse(
        String id,
        String title,
        String artist,
        List<String> tags,
        String thumbnailUrl,
        int viewCount
) {
}

File: src/main/kotlin/com/mymedia/streamer/dto/VideoCollectionResponse.kt
package com.mymedia.streamer.dto

/**
 * 비디오 컬렉션(폴더) 정보를 반환하는 응답 DTO
 */
data class VideoCollectionResponse(
    val id: String,
    val title: String,
    val artist: String,
    val tags: List<String>,
    val thumbnailUrl: String
)